.htmlvalidate.js

```diff
@@ -0,0 +1,21 @@
+const { defineConfig } = require("html-validate");
+
+module.exports = defineConfig({
+  extends: ["html-validate:recommended"],
+  rules: {
+    /* Define <!DOCTYPE html> to lowercase because Hugo minify does it */
+    "doctype-style": ["error", {"style": "lowercase"}],
+
+    /* Allow images to not have alt="" attribute */
```

Comment:
Why disable this?

Reply:
Up until now, we have chosen not to add an `alt` attribute if its value is null (see this [tutorial](https://www.w3.org/WAI/tutorials/images/decorative/) for decorative images).

Reply:
ok